datamade/committee-oversight
# Configure Sentry DSN via local_settings

## Overview

This PR removes the live Sentry DSN from the main settings file, and uses `local_settings.py` to 

## Testing Instructions

 * Run the app locally and do something that raises an exception, e.g., remove all committee ratings data in the Django shell, then visit the homepage.
 * Confirm that you see an exception in your browser, but no exception is logged to Sentry or in Slack.
